Documentation/gettingstarted/k8s-install-eks.rst

```diff
@@ -90,6 +90,20 @@ Deploy Cilium release via Helm:
      --set global.tunnel=disabled \\
      --set global.nodeinit.enabled=true
 
+.. note::
+
+   This helm command sets ``global.eni=true`` and ``global.tunnel=disabled``,
+   meaning that Cilium will allocate a fully-routable AWS ENI IP address for each pod,
+   similar to the behavior of the
+   `Amazon VPC CNI plugin <https://docs.aws.amazon.com/eks/latest/userguide/pod-networking.html>`_.
+
+   Cilium can also run in EKS using an overlay mode that gives pods non-VPC-routable IPs.
```

Comment:
```suggestion
   Cilium can alternatively run in EKS using an overlay mode that gives pods non-VPC-routable IPs.
```

Reply:
Good suggestion.  Just pushed this change.